### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by dryajov**

### CODE REVIEW

File: examples/directchat.nim [hunk 1]
```diff
@@ -49,8 +49,8 @@ type
 
 proc id (p: ChatProto): string =
   result = "unknown"
-  if p.conn.peerInfo.peerId.isSome:
-    result = $p.conn.peerInfo.peerId.get()
+  if p.conn.peerInfo.isSome:
```
Comment (arnetheduck) [inline examples/directchat.nim]: something to think of in compiled languages like nim: it's better not to do string assignments unnecessarily, so:
```
proc id(..): string = 
  if p.conn.peerInfo.isSome: ... 
  else: "unknown"
```

Comment (dryajov) [reply examples/directchat.nim]: Is there really a big difference with regards to interpreted languages when it comes to allocations?

In general, I've seen advocates of this style on both sides of the isle - interpreted and compiled. The argument for it is that it makes the code more readable and less error prone; against, as you imply, the _potentialy_ redundant allocation. 

I think you can argue the pros and cons of this style until the end of times and end up right where you started - as it usually happens with this sort of things. Personally, I've gotten quite used to it and I do find it more readable and less error prone, in addition moder optimizers are good enough to spot optimize it out altogether - this is also partially confirmed by inspecting the generated C code, the string is nowhere to be found.

That said, I'm neither agreeing nor disagreeing - IMO this is a matter of style/taste.

Comment (zah) [reply examples/directchat.nim]: The Nim compiler in particular doesn't have any optimisation pass that will help here. The C compiler may be able to figure it out, but considering that copying a string invokes the side-effects of the entire memory allocator (this includes potentially calling an OS routine such as `mmap`), the compiler should be shy about making this transformation.

Comment (dryajov) [reply examples/directchat.nim]: Ok, seems like the safest thing to do here is to avoid this style.

File: .travis.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ install:
 
   # install and build go-libp2p-daemon
   - curl -O -L -s -S https://raw.githubusercontent.com/status-im/nimbus-build-system/master/scripts/build_p2pd.sh
-  - bash build_p2pd.sh p2pdCache HEAD
+  - bash build_p2pd.sh p2pdCache v0.0.1
```
Comment (stefantalpalaru) [inline .travis.yml]: Don't you want "v0.2.1" in here? That's the go-libp2p-daemon version we use in nim-beacon-chain.

Comment (dryajov) [reply .travis.yml]: I'm playing it safe, but perhaps you're right, we can bump it to a higher version.

**Pull request merged**